posydon/binary_evol/DT/step_detached.py

```diff
@@ -1038,9 +1033,6 @@ def __call__(self, binary):
             elif (binary.star_1.state in LIST_ACCEPTABLE_STATES_FOR_HeStar):
                 secondary.htrack = False
             elif (binary.star_1.state in STAR_STATES_CO):
-                binary.state += " Thorne-Zytkow object"
-                if self.verbose or self.verbose == 1:
-                    print("Formation of Thorne-Zytkow object, nothing to do further")
```

Comment:
@ezapartas , please check this change. You have introduced this code part.
I'd agree, Thorne-Zytkow objects shouldn't be introduced in the detached but in the merger step. What was the reason, why you put it in here instead of the merger step?
Surely we shouldn't simply delete this code it should be moved to the correct place (I think this should be in the merger step).

Reply:
We've discussed this more in the issue related to this #195.

@ezapartas do you want to add it in the merger step? If so, can you add that to this PR?